### DISCUSSION TITLE
ImportError for mask_to_index

### DISCUSSION THREAD

**Discussion by JiaruiWang** [pyg-team/pytorch_geometric#4734] [Q&A]
The model can run successfully. However, I got the error when I try to import ```mask_to_index```. 
```from torch_geometric.utils import index_to_mask, mask_to_index```
```ImportError: cannot import name 'mask_to_index' from 'torch_geometric.utils'```
The ```index_to_mask``` can be imported.

**Comment by rusty1s** [comment]
We just recently added this function. If you want to use it, you have to install PyG from master/nightly, see [here](https://github.com/pyg-team/pytorch_geometric#nightly-and-master). Alternatively, you can just implement the functionality on your end. It is not too hard to do so, see [here](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/utils/mask.py#L22-L28).
